lib/cartopy/mpl/ticker.py

```diff
@@ -274,10 +281,16 @@ def __init__(self, degree_symbol='\u00B0', number_format='g',
             auto_hide=auto_hide,
         )
 
+        self._direction_labels = direction_label
+
     def _apply_transform(self, value, target_proj, source_crs):
         return target_proj.transform_point(0, value, source_crs)[1]
 
     def _hemisphere(self, value, value_source_crs):
+
+        if not self._direction_labels:
```

Comment:
What about moving this up to the formatting base class? Only call `self._hemisphere()` if necessary

Up above, something like this.
`hemisphere = self._hemisphere() if self._direction_labels else ''`

Reply:
This would mean adding the `_direction_labels` argument to `_PlateCarreeFormatter` and removing it from the other classes where I've just added it, correct ?

Reply:
I think it would just mean adding it to the _PlateCarreeFormatter as an additional option there too, you'd still want the explicit keyword arguments in this function I think.